Assets/LeapMotionModules/Hands/Scripts/RiggedHand.cs

```diff
@@ -6,25 +6,55 @@
 
 using UnityEngine;
 using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
 using Leap;
 
 namespace Leap.Unity {
-  // Class to setup a rigged hand based on a model.
+  /** This version of IHandModel supports a hand respresentation based on a skinned and jointed 3D model asset.*/
   public class RiggedHand : HandModel {
     public override ModelType HandModelType {
       get {
         return ModelType.Graphics;
       }
     }
     public override bool SupportsEditorPersistence() {
-      return true;
+      return SetEditorLeapPose;
     }
+    [Tooltip("When True, hands will be put into a Leap editor pose near the LeapServiceProvider's transform.  When False, the hands will be returned to their Start Pose if it has been saved.")]
+    [SerializeField]
+    private bool setEditorLeapPose = true;
+    
+    public bool SetEditorLeapPose {
+      get { return setEditorLeapPose; }
+      set {
+
+        if (value == false) {
+          RestoreJointsStartPose();
+        }
+        setEditorLeapPose = value;
+      }
+    }
+
     [Tooltip("Hands are typically rigged in 3D packages with the palm transform near the wrist. Uncheck this is your model's palm transform is at the center of the palm similar to Leap's API drives")]
     public bool ModelPalmAtLeapWrist = true;
+    [Tooltip("Set to True if each finger has an extra trasform between palm and base of the finger.")]
     public bool UseMetaCarpals;
     public Vector3 modelFingerPointing = new Vector3(0, 0, 0);
     public Vector3 modelPalmFacing = new Vector3(0, 0, 0);
+    [Header("Values for Stored Start Pose")]
+    [SerializeField]
+    private List<Transform> jointList = new List<Transform>();
+    [SerializeField]
+    private List<Quaternion> localRotations = new List<Quaternion>();
+    [SerializeField]
+    private List<Vector3> localPositions = new List<Vector3>();
 
+    void OnValidate() {
+      if (SetEditorLeapPose == setEditorLeapPose) {
```

Comment:
Not sure what is going on here?  SetEditorLeapPose is just a public getter for the private setEditorLeapPose